test.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * Comment
+ */
```

Comment:
🚫 **[phpcs]** <Squiz.Commenting.FileComment.MissingPackageTag> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing @package tag in file comment